pkg/conversions/virtualrouter_types_conversion.go

```diff
@@ -98,8 +70,38 @@ func Convert_CRD_HTTPRouteMatch_To_SDK_HttpRouteMatch(crdObj *appmesh.HTTPRouteM
 	sdkObj.Method = crdObj.Method
 	sdkObj.Prefix = aws.String(crdObj.Prefix)
 	sdkObj.Scheme = crdObj.Scheme
+
+	if crdObj.Path != (appmesh.HTTPPathMatch{}) {
```

Comment:
Aren't we comparing crdObj to appmesh type here? Shouldn't this be appmeshsdk.HTTPPathMatch instead?

Reply:
This checks if HTTPPathMatch struct is specified by the user, only then we try to convert it